<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainControlPanel</class>
 <widget class="QWidget" name="MainControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>715</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>191</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>191</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
QGroupBox {
  background-origin: content;
  margin-top: 15px;
  font-weight: bold;
  font-size: 12px;
  color: rgb(30,30,160);
  background-color: rgb(237,237,237);
  padding: 5px;
  border-radius: 4px;
  border: 1px solid rgb(130,130,130);
}
QGroupBox::title {
  subcontrol-origin: 	margin;
  subcontrol-position: top left;
} */

QGroupBox {
  background-origin: content;
  margin-top: 15px;
  font-weight: bold;
  font-size: 12px;
  text-shadow:1px 1px 0 white;
  color: rgb(30,30,160);
  background-color: rgb(222,222,222);
  padding: 5px;
  border-radius: 2px;
  border: 1px solid rgb(168,168,168);
}
QGroupBox::title {
  subcontrol-origin: 	margin;
  subcontrol-position: top left;
}

QToolButton::checked {
  background-color:rgb(210, 210, 210);
}

QToolButton::pressed {
  background-color:rgb(210, 210, 210);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="grpToolbox">
        <property name="styleSheet">
         <string notr="true">
QToolBar {
	padding: 0px;
}</string>
        </property>
        <property name="title">
         <string>Main Toolbar</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="panelToolbarMode" native="true">
           <property name="styleSheet">
            <string notr="true">QToolButton {
	width: 22px;
	height: 22px;
	icon-size: 22px;
	padding: 1px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="panelToolbarAction" native="true">
           <property name="styleSheet">
            <string notr="true">QToolButton {
	width: 16px;
	height: 16px;
	icon-size: 16px;
	padding: 1px;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpInspector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Cursor Inspector</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">QToolButton {
  icon-size: 22px;
  min-width: 1ex;
  border-top: 1px solid rgb(192, 192, 192);
  border-bottom: 1px solid rgb(192, 192, 192);
  border-left: 1px;
  border-right: 1px;
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.0394089 rgba(235, 235, 235, 255), stop:0.487685 rgba(216, 216, 216, 255), stop:0.502463 rgba(205, 205, 205, 255), stop:1 rgba(235, 235, 235, 255));
  width: 25px;
  height: 25px;
}
QToolButton:hover, QToolButton:checked {
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.0394089 rgba(214, 228, 245, 255), stop:0.487685 rgba(193, 205, 221, 255), stop:0.502463 rgba(182, 193, 207, 255), stop:1 rgba(215, 228, 245, 255));
  padding:0px;
}
QWidget#widget {
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.0394089 rgba(235, 235, 235, 255), stop:0.487685 rgba(216, 216, 216, 255), stop:0.502463 rgba(205, 205, 205, 255), stop:1 rgba(235, 235, 235, 255));
  border-top: 1px solid rgb(192, 192, 192);
  border-bottom: 1px solid rgb(192, 192, 192);
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="btnCursorInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Cursor Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Examine the voxel under the cursor and enter exact 3D cursor location.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/crosshair.gif</normaloff>:/root/crosshair.gif</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnZoomInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>30</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Zoom Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Control zoom factor in all slice views. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/zoom.gif</normaloff>:/root/zoom.gif</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnDisplayInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Display Layout Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Control which slice views are shown in the main ITK-SNAP window and how multiple image layers are rendered. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/dl_toolbox.png</normaloff>:/root/dl_toolbox.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnSyncInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Synchronization Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Adjust synchronization of cursor, zoom and pan across multiple ITK-SNAP sessions running on one computer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/network-wireless.png</normaloff>:/root/network-wireless.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnPolygonInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Synchronization Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Adjust synchronization of cursor, zoom and pan across multiple ITK-SNAP sessions running on one computer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/poly.gif</normaloff>:/root/poly.gif</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnPaintbrushInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Synchronization Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Adjust synchronization of cursor, zoom and pan across multiple ITK-SNAP sessions running on one computer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/paintbrush.gif</normaloff>:/root/paintbrush.gif</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnSnakeInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Synchronization Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Adjust synchronization of cursor, zoom and pan across multiple ITK-SNAP sessions running on one computer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/roi.gif</normaloff>:/root/roi.gif</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnJoinInspector">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Join Inspector&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Choose mode for Join tool.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../Resources/SNAPResources.qrc">
                <normaloff>:/root/globalWS.gif</normaloff>:/root/globalWS.gif</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stack">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="CursorInspector" name="pageCursorInspector"/>
           <widget class="ZoomInspector" name="pageZoomInspector"/>
           <widget class="DisplayLayoutInspector" name="pageDisplayInspector">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
           <widget class="SynchronizationInspector" name="pageSyncInspector">
            <property name="toolTip">
             <string/>
            </property>
           </widget>
           <widget class="SnakeToolROIPanel" name="pageSnakeTool"/>
	   <widget class="JoinDataPanel" name="pageJoinTool"/>
           <widget class="PaintbrushToolPanel" name="pagePaintbrushTool"/>
           <widget class="PolygonToolPanel" name="pagePolygonTool"/>
           <widget class="QWidget" name="pageBlank"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpToolbox3D_2">
        <property name="styleSheet">
         <string notr="true">

QToolBar {
	padding: 0px;
}</string>
        </property>
        <property name="title">
         <string>Segmentation Labels</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="LabelInspector" name="labelInspector" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
	width: 22px;
	height: 22px;
	icon-size: 22px;
	padding: 1px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpToolbox3D">
        <property name="styleSheet">
         <string notr="true">

QToolBar {
	padding: 0px;
}</string>
        </property>
        <property name="title">
         <string>3D Toolbar</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="panelToolbarMode3D" native="true">
           <property name="styleSheet">
            <string notr="true">QToolButton {
	width: 22px;
	height: 22px;
	icon-size: 22px;
	padding: 1px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CursorInspector</class>
   <extends>QWidget</extends>
   <header>CursorInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZoomInspector</class>
   <extends>QWidget</extends>
   <header>ZoomInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelInspector</class>
   <extends>QWidget</extends>
   <header>LabelInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SnakeToolROIPanel</class>
   <extends>QWidget</extends>
   <header>SnakeToolROIPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaintbrushToolPanel</class>
   <extends>QWidget</extends>
   <header>PaintbrushToolPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DisplayLayoutInspector</class>
   <extends>QWidget</extends>
   <header>DisplayLayoutInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SynchronizationInspector</class>
   <extends>QWidget</extends>
   <header>SynchronizationInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JoinDataPanel</class>
   <extends>QWidget</extends>
   <header>JoinDataPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PolygonToolPanel</class>
   <extends>QWidget</extends>
   <header>PolygonToolPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/SNAPResources.qrc"/>
 </resources>
 <connections/>
</ui>
